app/src/main/java/org/piramalswasthya/sakhi/configuration/EligibleCoupleTrackingDataset.kt

```diff
@@ -190,37 +195,147 @@ class EligibleCoupleTrackingDataset(
 
             isPregnancyTestDone.id -> {
                 isPregnant.isEnabled = true
-                triggerDependants(
-                    source = isPregnancyTestDone,
-                    passedIndex = index,
-                    triggerIndex = 0,
-                    target = pregnancyTestResult
-                )
+
+                if (isPregnant.value==resources.getStringArray(R.array.yes_no_donno)[0])
+                {
```

Comment:
Follow same pattern with curly braces? Some places it's in new line, some places it's with the if condition.